diagnostics-with-elk/azuredeploy.json

```diff
@@ -123,17 +123,12 @@
             "type": "string",
             "defaultValue": "14.04.2-LTS",
             "allowedValues": [
-                "12.04.2-LTS",
-                "12.04.3-LTS",
-                "12.04.4-LTS",
-                "12.04.5-LTS",
                 "12.10",
```

Comment:
Is there a way to determine what is supported? On the Azure portal I still see 12.04.5 LTS, 15.10. I just need to list the Azure supported versions there. Maybe there should be a ARM template helper function?